plugins/inputs/aerospike/aerospike.go

```diff
@@ -161,6 +161,60 @@ func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) erro
 			}
 			acc.AddFields("aerospike_namespace", nFields, nTags, time.Now())
 		}
+
+		latency, err := as.RequestNodeInfo(n, "latency:")
+		if err != nil {
+			return err
+		}
+		result, exists := latency["latency:"]
+		if !exists {
+			return errors.New("Aerospike Input Plugin : No key `latency:` in response of as.RequestNodeInfo(n, \"latency:\") from Aerospike Server")
+		}
+		points := strings.Split(result, ";")
+		i := 0
+		for i < len(points) {
+			point := points[i]
+			if strings.Compare(point, "error-no-data-yet-or-back-too-small") == 0 {
+				i++
+			} else {
+				indexOfColon := strings.Index(point, ":")
+				if indexOfColon == -1 {
+					i++
+					continue
+				}
+				histogram := point[0:indexOfColon]
+				operation, namespace := splitNamespaceAndOperation(histogram)
+				latencyKey := point[indexOfColon:]
+				latencyValue := points[i+1]
+				keys := strings.Split(latencyKey, ",")
+				values := strings.Split(latencyValue, ",")
+				ltags := map[string]string{
+					"aerospike_host": hostport,
+					"node_name":      n.GetName(),
+					"namespace":      namespace,
+				}
+				lfields := make(map[string]interface{})
+				var duration, totalOperations float64
+				duration = getDurationOfTransaction(keys[0], values[0])
+				operationsPerSecond, err := parseValue(values[1])
```

Comment:
Verify the length of values > 0 so this line doesn't panic

Reply:
Done.